CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased]
 ### Added
 - [\#131](https://github.com/Manta-Network/manta-rs/pull/131) Add abstract Phase 1 for Groth16 trusted setup
+- [\#168](https://github.com/Manta-Network/manta-rs/pull/168) Deprecate old Poseidon hash
```

Comment:
This should be in `Removed` instead of `Added` even though we did do two things in this PR, the remove is more important than what we added.